tools/watermill-cli/README.md

```diff
@@ -0,0 +1,62 @@
+# CLI Wrapper for Watermill
+
+This is a CLI wrapper for Watermill. It has two basic functionalities: producing and consuming messages on the following
```

Comment:
Maybe:
> Simple CLI tool for consuming and producing messages, similar to Kafka Console Producer/Consumer  Works with:
?

Reply:
I'll work on that

Reply:
b3ae9f252ccb6e90efcf9f72512765a9df71cff4 rephrased this